content/en/docs/migration/opentracing.md

```diff
@@ -179,16 +170,20 @@ context manager. Using both methods within the same trace may create broken or
 mismatched spans, and is not recommended.
 
 Instead of mixing the two APIs in the same trace, we recommend that you migrate
-complete codepaths from OpenTracing to OpenTelemetry as a single unit, so that
+complete code paths from OpenTracing to OpenTelemetry as a single unit, so that
 only one API is used at a time.
 
 
 ## Specification and implementation details
 
-For details on how each individual OpenTracing shim works, please see the
-appropriate language-specific documentation. For details on the design of the
-OpenTracing shim, please see the
-[specification].
+For details on how each individual OpenTracing shim works, see the appropriate
+language-specific documentation. For details on the design of the OpenTracing
+shim, see [OpenTracing Compatibility][OT_spec].
 
+[.NET]: https://opentelemetry.io/docs/instrumentation/net/shim/
+[Go]: https://pkg.go.dev/go.opentelemetry.io/otel/bridge/opentracing
+[Java]: https://github.com/open-telemetry/opentelemetry-java/tree/main/opentracing-shim
+[Javascript]: https://www.npmjs.com/package/@opentelemetry/shim-opentracing
 [OpenTracing]: https://opentracing.io
-[specification]: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/compatibility/opentracing.md
+[OT_spec]: {{< relref "/docs/reference/specification/compatibility/opentracing" >}}
+[Python]: https://opentelemetry-python.readthedocs.io/en/stable/shim/opentracing_shim/opentracing_shim.html
```

Comment:
For some reason the Python link looks broken when viewing the rendered markdown?

Reply:
I'm not sure why, but Hugo [renders it just fine](https://deploy-preview-1090--opentelemetry.netlify.app/docs/migration/opentracing/#language-version-support).